db/migrate/20251027103851_add_fields_to_image_data.rb

```diff
@@ -0,0 +1,12 @@
+class AddFieldsToImageData < ActiveRecord::Migration[8.0]
+  def change
+    safety_assured do
+      change_table :image_data, bulk: true do |t|
+        t.json :crop_data
+        t.json :dimensions
```

Comment:
Would be good to have an idea of what the json looks like for both of the above (can just be in the commit message), which would explain why we're not setting individual properties (e.g. `offset_x`) in the DB.